db/migrate/20200727113800_change_va_caution_flag_date_type.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeVaCautionFlagDateType < ActiveRecord::Migration[4.2]
+    def change
+        drop_table :va_caution_flags
+          create_table :va_caution_flags do |t|
```

Comment:
Is the tabulation off here? I'm a bit confused.

Reply:
The tabulation is off.

Reply:
I've fixed it.